src/app/static/src/app/components/plots/Choropleth.vue

```diff
@@ -77,18 +84,10 @@
                 }
             },
             prevEnabled: function() {
-                const result = !!(this.indicatorData.prevRange.min || this.indicatorData.prevRange.max);
```

Comment:
Trying to derive whether a indicator should be enabled based on the range of the data returned isn't really going to work because it's now perfectly valid for there to be no data points in the data being displayed as the combination of selected filters may not map to any values (this is often the case in the sample data). Hardcoding it by dataType for now, rather than doing a separate sweep of the unfiltered data to see whether there are any values for the indicator. Suspect we'll revisit this at some point, might end up being metadata.